transport/http/constants.go

```diff
@@ -22,20 +22,24 @@ package http
 
 const (
 	// CallerHeader is the HTTP header used to indiate the service doing the calling
-	CallerHeader = "RPC-Caller"
+	CallerHeader = "Rpc-Caller"
```

Comment:
HTTP is case-insensitive and so in that way it's the "lowest common denominator" that we need to support. I just want to confirm that if someone gives us a curl statement with "RPC-Caller" we will do the right thing.